src/panels/config/devices/device-detail/integration-elements/zwave_js/ha-device-info-zwave_js.ts

```diff
@@ -116,6 +116,14 @@ export class HaDeviceInfoZWaveJS extends LitElement {
           ? this.hass.localize("ui.common.yes")
           : this.hass.localize("ui.common.no")}
       </div>
+      <div>
+        ${this.hass.localize(
+          "ui.panel.config.zwave_js.device_info.zwave_plus"
+        )}:
+        ${this._node.zwave_plus_version
+          ? this.hass.localize("ui.common.yes")
+          : this.hass.localize("ui.common.no")}
+      </div>
```

Comment:
Would be preferable to show the actual Z-Wave Plus version, somethingg "No", "1" or "2".